requests/requests.py

```diff
@@ -0,0 +1,106 @@
+TOKEN = ''
```

Comment:
Your pull request: REQUESTS-1021 Added requests.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: REQUESTS-1021 Added requests.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: REQUESTS-1021 Added requests-with-checkihg-time.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: Delete requests-with-checkihg-time.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Your commit: REQUESTS-1021 Refactored requests.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']
Your commit: Merge remote-tracking branch 'origin/requests' into requests
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Action of your comment must be from ['Added', 'Deleted']
Your commit: REQUESTS-1021 Refactored requests.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']